R/zchunk_LA116.geo.R

```diff
@@ -1,23 +1,27 @@
 #' module_energy_LA116.geo
 #'
-#' Briefly describe what this chunk does.
+#' Generate geothermal (hydrothermal and engineered geothermal systems (EGS)) supply curves by GCAM region
+#' Supply curves developed for the 14 GCAM 3.0 regions are downscaled to the country level on the basis
+#' of land area, and aggregated to the current GCAM regions. The data source for the supply curves is
+#' Hannam et al. 2009: http://www.pnl.gov/main/publications/external/technical_reports/PNNL-19231.pdf
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L116.RsrcCurves_EJ_R_geo}, \code{L116.RsrcCurves_EJ_R_EGS}. The corresponding file in the
 #' original data system was \code{LA116.geo.R} (energy level1).
-#' @details Describe in detail what this chunk does.
+#' @details This code chunk assigns geothermal electric resources to GCAM regions. Because the underlying
+#' source data were developed for the 14 regions of GCAM 3.0 and prior, they are first
+#' downscaled to the nation level on the basis of land area, and aggregated to the current GCAM regions.
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
-module_energy_LA116.geo_DISABLED <- function(command, ...) {
+#' @author GPK April 2017
+module_energy_LA116.geo <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "common/iso_GCAM_regID",
-             "L100.Land_type_area_ha",
+             FILE = "aglu/LDS/Land_type_area_ha",
```

Comment:
That's correct. It's the output of Alan's Land Data System but it's just a versioned file treated as data from an external source (the LDS weighs in at something like 40 GB so it's not going to be part of the data system)